pkg/controller/controller.go

```diff
@@ -480,10 +480,13 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 			}
 		}
 
+		ns, _ := c.kubeClient.Core().Namespaces().Get(instance.Namespace)
```

Comment:
Seems to me that if there is no namespace then we should stop and error out. Sending a request to a broker referencing the GUID of a namespace that doesn't exist is asking for trouble if that Broker ever tries to contact the platform looking for the NS.  Plus,when the NS does get created (if it does) then it'll have a new GUID that won't match - causing even more confusion.   No?